inc/saiacl.h

```diff
@@ -221,11 +221,15 @@ typedef enum _sai_acl_table_attr_t
     
     /** READ-ONLY */
 
+    /** Switch Object ID [sai_object_id_t] (CREATE_ONLY),
+     * Default SAI_NULL_OBJECT_ID */
```

Comment:
Maybe its more convinient to use SAI_NULL_OBJECT_ID as default switch id, but there should be api that will get default switch ID on sai_switch_api->get_switch_attribute(0, 1, &attr) where attr.id = SAI_SWITCH_DEFAULT_SWITCH_ID; since when multiple switches are present there should be way to get their id, and query type of that object should be SAI_OBJECT_TYPE_SWITCH; 

IMHO SAI_NULL_OBJECT_ID should only be used to disable some functionality that uses object id, instead of pointing to "default" instance of some object. A good example here is DEFAULT_VIRTUAL_ROUTER, when creating ROUTE we don't set vr_id to NULL but to actual object id, this helps to track object dependencies without any special corner cases logic

So for all objects Default should be to SAI_SWITCH_DEFAULT_SWITCH_ID instead of SAI_NULL_OBJECT_ID 

only inconvinience here is that 
 sai_switch_api->get_switch_attribute(0, 1, &attr)  uses "zero" to get default_switch_id

also IMHO since we are moving to sai_switch_create() api, there should be no default switch, user should be obligated to create it and then get proper switch id, then all that "default" switch id would be gone